packages/wallet/src/redux/protocols/existing-ledger-funding/__tests__/reducer.test.ts

```diff
@@ -67,10 +66,9 @@ describe('player B happy path', () => {
   });
 
   describeScenarioStep(scenario.waitForLedgerUpdate, () => {
-    const { state, action, sharedData, reply } = scenario.waitForLedgerUpdate;
+    const { state, action, sharedData } = scenario.waitForLedgerUpdate;
     const updatedState = existingLedgerFundingReducer(state, sharedData, action);
     itTransitionsTo(updatedState, 'ExistingLedgerFunding.Success');
-    itSendsMessage(updatedState, reply);
```

Comment:
Now that we're using the consensus update protocol we can't really expect a certain reply in our tests. This is due to the fact that Consensus Update will reply with whatever its success commitment is which can differ from what we expect here.

We could just check that some message is sent out or start exporting a `successReply` from Consensus Update?

Reply:
I think exporting a `successReply` is a better idea than not testing it.